documentation/src/docs/asciidoc/release-notes/release-notes-5.8.0-M1.adoc

```diff
@@ -86,6 +86,9 @@ on GitHub.
 * In parameterized tests with `@MethodSource` or `@ArgumentSource`, arguments can now have
   optional names. When the argument is included in the display name of an iteration, this
   name will be used instead of the value.
+* `DynamicTests.stream()` can now consume named inputs, and will use those names as test
+  names.
```

Comment:
```suggestion
* `DynamicTests.stream()` can now consume `Named` input and will use each name-value
   pair as the display name and value for each generated dynamic test.
```

Reply:
Probably good to link either to the Javadoc for `Named` or to a section in the User Guide covering the topic.

Reply:
You mean link as in clickable link ?

Reply:
Or just a mention?

Reply:
Yes, I mean a hyperlink.

Reply:
You added new examples in the `documentation` project. So you should be able to link to the section of the User Guide that displays those examples.

Reply:
But do we usually put links in the release notes?

Reply:
Yes, when a new feature is introduced we often link to the corresponding section of the User Guide if such a section exists.

Typically it takes the form:

* Introduced feature XYZ
  * See `<link>` in the User Guide for details.

Reply:
Although it appears that we have not provided such links in recent releases, here's an example where we did: https://junit.org/junit5/docs/5.5.0/release-notes/index.html#new-features-and-improvements-3

> Support for declarative timeouts using `@Timeout` or configuration parameters (see **User Guide** for details).

Reply:
Not sure I did it the right asciidoc-way 😄